OpenRA.Game/Network/OrderManager.cs

```diff
@@ -194,6 +213,13 @@ public void Tick()
 			foreach (var order in frameData.OrdersForFrame(World, NetFrameNumber))
 				UnitOrders.ProcessOrder(this, World, order.Client, order.Order);
 
+			if (World.PlayerWinStates != playerWinstates)
+			{
+				// Sync Hash and Player Win States share the same sync requirement
+				Connection.SendSync(NetFrameNumber, OrderIO.SerializeWinState(World.PlayerWinStates));
```

Comment:
What about renaming the method to `SendPacketForFrame`? And the order based `Send` method could be renamed to `SendOrdersForFrame`.

Reply:
I'm fine with that; do you want this change in this PR (separated commit) or it can wait?